.github/workflows/test.yml

```diff
@@ -64,8 +64,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-      #   feature: [default-features, no-default-features, dns-over-rustls, dns-over-https-rustls, dns-over-native-tls, dns-over-openssl, dnssec-openssl, dnssec-ring, mdns, async-std]
-        feature: [default-features, no-default-features, dns-over-rustls, dns-over-https-rustls, dns-over-native-tls, dnssec-openssl, dnssec-ring, mdns, async-std]
+        feature: [default-features, no-default-features, dns-over-rustls, dns-over-https-rustls, dns-over-native-tls, dns-over-openssl, dnssec-openssl, dnssec-ring, mdns, async-std]
```

Comment:
Looks like there are still a bunch of inadvertent changes here compared to main (looking like you rolled back some changes to an earlier parent)?

Reply:
Muscle memory. I rebased against `origin/master` :facepalm:

Reply:
Makes sense. I just deleted the master branch in origin to prevent this kind of thing from happening.

Reply:
Thanks. I should have done that.